spec/openapi.yaml

```diff
@@ -1,44 +1,140 @@
 openapi: 3.0.1
+
+servers:
+- url: https://resources.operationcode.org/api/v1
+  description: Default Server
 info:
   title: Operation Code Resources API
-  description: A collection of programming and cyber security learning resources for
-    the Operation Code community.
+  version: '1.0'
+  x-logo:
+    url: 'https://operation-code-assets.s3.us-east-2.amazonaws.com/branding/logos/small-blue-logo.png'
+    altText: 'Operation Code Logo'
+    backgroundColor: '#252e3e'
+  description: |
+    # Introduction
+    Operation Code Resources API is a collection of programming and cyber security learning resources curated by and for the Operation Code community.
+
+    # Authentication
+    Create (POST) and update (PUT) routes are protected and require an API key. API key requests to the `/apikey` endpoint require valid Operation Code credentials in the request body. 
+    
+    If you are not already a member of our community, join Operation Code [HERE](https://https://operationcode.org/join).
```

Comment:
This link is broken. I think probably because of the double `https://` thing. In my browser it redirects to https://https//operationcode.org/join which is a 404

Reply:
Err, not even a 404, ERR_NAME_NOT_RESOLVED

Reply:
Good catch! Removed duplicate and will push up with other changes.

Reply:
It's weird, it says "outdated" but as of commit 097a5374 I still see the double https declaration. 🤔

Reply:
Somehow I had outdated code locally. I'm not really sure what happened. Disregard this comment, and if any other comments are incorrect on the latest code, disregard those as well, though I think all of the rest still apply